AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py

```diff
@@ -22,6 +22,12 @@ class Tests:
     cubemap_property_set = (
         "Cubemap property set on HDRi Skybox component",
         "Couldn't set Cubemap property on HDRi Skybox component")
+    exposure_set_to_negative_five = (
+        "Exposure set to negative 5",
+        "Exposure could not be set to negative 5")
```

Comment:
spelling out negative seems odd also is this a float so -5.0

Reply:
fixed